src/main/java/com/theelixrlabs/healthcare/exceptionHandler/CustomException.java

```diff
@@ -0,0 +1,10 @@
+package com.theelixrlabs.healthcare.exceptionHandler;
+
+/**
+ * Custom Exception class for handling exception.
+ */
+public class CustomException extends RuntimeException{
+    public CustomException(String ExceptionMessage) {
```

Comment:
ExceptionMessage should be in camel case

exceptionMessage

Reply:
Yes I changed to exceptionMessage.